bin/create-ssl-certificate.js

```diff
@@ -104,16 +109,16 @@ ${colors.white}
   1. Drag and drop the created .crt file into Keychain Access -> Certificates
   2. Double click added certificate -> Trust section
   3. Change to always trust
-`);
+`)
   return pause()
 })
-.catch(logAndAbort)
+.catch(abort)
```

Comment:
This is mainly for supporting `CTRL-C` sigint to prevent opening keychain and `open .`. So not an error per se. So I changed it to `process.exit(0)`